### ISSUE TITLE
Mulligan should automatically activate

### ISSUE THREAD

**Issue by bazola** [Cardshifter/Cardshifter#146]
Rather than having to press a button to initiate the Mulligan at the start of the game, it should activate automatically once it is available

**Comment by Zomis** [comment]
Should this perhaps be flexibilized so that whenever there is only one action to perform, that action button is automatically 'clicked'? For example: If you can do nothing, automatically press end turn.

**Comment by Phrancis** [comment]
I'm not sure from a UX perspective this would be ideal @Zomis - Think of HS or even a regular card game, I think a player should be able to realize on their own that they can't play anything... At least something like HS where you see the green highlight on "End Turn" that tells a user they can't play anything further.

**Comment by Zomis** [comment]
Good point, @Phrancis. I'm not sure how @bazola plans on implementing this, but from the description it sounded like it would be a `if (actionName.equals('Mulligan'))` which I'm not sure how I feel about.
